src/components/InstanceMonitoring/InstanceMonitoringContent.js

```diff
@@ -0,0 +1,46 @@
+import React, { Component } from 'react';
+import { InstanceMonitoringMetric } from './InstanceMonitoringMetric';
+
+export class InstanceMonitoringContent extends Component {
+  render() {
+    let metricInfo = [
+      {
+        key: '1',
+        amount: '3',
+        unit: 'mbps',
+        desc: 'Throughput',
+      },
+      {
+        key: '2',
+        amount: '10',
+        unit: 'cps',
+        desc: 'Connection rate',
+      },
+      {
+        key: '3',
+        amount: '2',
+        unit: 'B',
+        desc: 'Data processed this month',
+      },
+      {
+        key: '4',
+        amount: '3',
+        unit: '',
+        desc: 'Active connections',
+      },
+    ];
+
+    let metrics = metricInfo.map(metric => {
```

Comment:
Does it make sense to make this variable `const`, so that nobody re-assigns to this variable by mistake?